version.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"fmt"
+	"runtime"
+)
+
+// populated by -ldflags from Makefile
+var version = "unknown"
```

Comment:
I thought about that, but it would make it difficult for people using `go run` etc...  I'm not sure this is something that would be a problem in reality.  The builds where this matters are the ones we deploy, and they'll always be built on the CI servers using make.